src/mrack/transformers/beaker.py

```diff
@@ -97,5 +147,60 @@ def create_host_requirement(self, host):
             "arch": host.get("arch", "x86_64"),
             "variant": variant,
             "ks_meta": self._get_ks_meta(host),
+            "retention_tag": self._find_value(
+                host.get(CONFIG_KEY, {}),
+                "retention_tag",
+                "retention_tag",
+                host["os"],
+                default="audit",
+            ),
+            "product": self._find_value(
+                host.get(CONFIG_KEY, {}),
+                "product",
+                "product",
+                host["os"],
+                default="[internal]",
+            ),
+            "whiteboard": self._find_value(
+                host.get(CONFIG_KEY, {}),
+                "whiteboard",
+                "whiteboard",
+                host["os"],
+                default="This job has been created using mrack.",
+            ),
+            "priority": self._find_value(
+                host.get(CONFIG_KEY, {}),
+                "priority",
+                "priority",
+                host["os"],
+                default="Normal",
+            ),
+            # Recipe task definition
+            "tasks": self._find_value(
+                host.get(CONFIG_KEY, {}),
+                "tasks",
+                "tasks",
+                host["os"],
+                default=[  # we use dummy task because beaker require a task in recipe
+                    {"name": "/distribution/dummy", "role": "STANDALONE"}
+                ],
+            ),
             f"mrack_{CONFIG_KEY}": host.get(CONFIG_KEY, {}),
         }
+
+        specs.update(
```

Comment:
Trivial (no need to change the PR if nothing else is changed):  Is there a need for `spec.update` ? It looks that it can be part of the dict definition.

Reply:
yes it can let me update

Reply:
Updated and commit with tests added